static/app/views/issueList/actions/headers.tsx

```diff
@@ -142,6 +217,17 @@ const PriorityLabel = styled(ToolbarHeader)<{isSavedSearchesOpen?: boolean}>`
   }
 `;
 
+const NarrowPriorityLabel = styled(IssueStreamHeaderLabel)`
+  display: flex;
+  justify-content: space-between;
+  width: 70px;
+
+  /* prettier-ignore */
```

Comment:
i'm not sure these are ignoring anything and the silly thing we use to process these does not remove comments at build time and they make it into production. might be worth not leaving these comments